src/Lean/Parser/Command.lean

```diff
@@ -80,7 +80,10 @@ def «example»        := leading_parser "example" >> ppIndent declSig >> declVa
 def ctor             := leading_parser "\n| " >> ppIndent (declModifiers true >> ident >> optDeclSig)
 def derivingClasses  := sepBy1 (group (ident >> optional (" with " >> Term.structInst))) ", "
 def optDeriving      := leading_parser optional (ppLine >> atomic ("deriving " >> notSymbol "instance") >> derivingClasses)
-def «inductive»      := leading_parser "inductive " >> declId >> optDeclSig >> optional (symbol " :=" <|> " where") >> many ctor >> optDeriving
+def computedField    := leading_parser ident >> " : " >> termParser >> Term.matchAlts
```

Comment:
I noticed this syntax doesn't support any declaration modifiers (e.g., documentation, visibility, attributes, etc.). Is this for expediency or are there real limitations to what kind of modifiers these fields can support?

Reply:
Thanks for the suggestion; docstrings, visibility, and attributes are supported now.